src/quantum/tp6-32bit.js

```diff
@@ -0,0 +1,89 @@
+import _ from 'lodash';
+import React from 'react';
+import { withStyles } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import { PAGES } from './config';
+import DashboardPage from '../components/DashboardPage/index';
+import PerfherderGraphContainer from '../containers/PerfherderGraphContainer/index';
+
+
+const styles = {
+  // This div helps with canvas size changes
+  // https://www.chartjs.org/docs/latest/general/responsive.html#important-note
+  body: {
+    backgroundColor: 'white',
+  },
+  area: {
+    display: 'flex',
+  },
+  column: {
+    flex: '50%',
+    alignItems: 'center',
+  },
+  chart: {
+    justifyContent: 'center',
+    padding: '1rem',
+
+  },
+
+};
+
+const TP6_32 = ({ classes }) => {
+  const allCharts = _
+    .chain(PAGES.data)
+    // ZIP HEADER WITH ROWS TO GET OBJECTS
+    .map(row => _.zipObject(PAGES.header, row))
+    // GROUP BY title
+    .groupBy(row => row.title)
+    // LOOP OVER EACH KEY/VALUE
+    .toPairs()
+    .map(([title, series], _i) => {
```

Comment:
If you're not using `_i` keep it out of the signature.

Reply:
this was garbage from my debugging